chrome/validate-nav.js

```diff
@@ -6,16 +6,16 @@ const navigationSchema = require('./validationSchemas/navigation');
 
 
 
-const navigationFiles = ['ansible', 'application-services', 'rhel', 'settings', 'user-preferences', 'openshift'].map((file) => path.resolve(__dirname, `${file}-navigation.json`))
+const navigationFiles = fs.readdirSync(__dirname)
+  .filter(f => f.endsWith('-navigation.json'))
+  .filter(f => f !== 'landing-navigation.json');
 const landingFile = path.resolve(__dirname, 'landing-navigation.json')
 const modulesFile = path.resolve(__dirname, 'fed-modules.json')
 
 
 async function validateLanding() {    
   try {
-    const file = fs.readFileSync(landingFile, {
-      encoding: 'utf-8'
-    })
+    const file = fs.readFileSync(landingFile, 'utf-8')
```

Comment:
This is completely unnecessary change. Both object and string works.

Reply:
Yes, but since I changed line 37 I decided to change them all to be consistent.